locales/en/errors.json

```diff
@@ -1,3 +1,14 @@
 {
-  "error-first-name-blank": "Enter the director’s full first name" 
+  "error-first-name-blank": "Enter the director’s full first name", 
+  "error-change-date-missing-day": "Date the director’s details changed must include a day",
+  "error-change-date-missing-month": "Date the director’s details changed must include a month",
+  "error-change-date-missing-year": "Date the director’s details changed must include a year",
+  "error-change-date-missing-day-month": "Date the director’s details changed must include a day and month",
+  "error-change-date-missing-day-year": "Date the director’s details changed must include a month and year",
+  "error-change-date-missing-day-month-year": "Enter the date the director’s details changed",
+  "error-change-date-invalid": "Date the director’s details changed must be a real date",
```

Comment:
Some of the text seems a little unfamiliar to me.  If this is what UX / Analysis have provided, fair enough.  But comparing with the GDS guide `https://design-system.service.gov.uk/components/date-input/#error-messages`
Should this be more like "Date of change must be a real date" ?
and below "Date of change must be the same as or after [appointment date], when the director was appointed"
This is more to ask if the GDS guide has been referred to.  Wouldn't affect my approving.

Reply:
Hey @PhilipGlover.
I will highlight these concerns to our BA who will discuss this with the UX/relevant team. 
Although regarding "_Date of change must be the same as or after [appointment date], when the director was appointed_" will require injecting the dynamic input in the error message, unsure if we can expose an actual appointment date.